CommonTools/RecoAlgos/interface/MVAComputer.h

```diff
@@ -0,0 +1,29 @@
+#ifndef COMMONTOOLS_RECOALGOS_MVACOMPUTER
+#define COMMONTOOLS_RECOALGOS_MVACOMPUTER
+#include <memory>
+#include <string>
+#include <vector>
+#include <tuple>
+
+#include <TMVA/Reader.h>
+
+class MVAComputer {
```

Comment:
this seems redundant with https://github.com/cms-sw/cmssw/blob/master/CommonTools/MVAUtils/interface/TMVAEvaluator.h

also, it's better to use the CMSSW custom GBRTree/Forest classes for BDT inference (if possible), as they are typically ~4x faster than TMVA

Reply:
@pmeridian @simonepigazzini Looks like this MVA uses Gradient Boosting and not AdaBoost? Just making sure.

Reply:
Hi, 

correct, the MVA uses gradient boosting. I was not aware of the existing interface, it should be trivial to migrate to it.

simone

Reply:
Does this migration require any change in the training file format?
If not, better switch to GBR forest now